### PR TITLE
Add new courses in PhD program

### PR DESCRIPTION
**Pull request by adamltyson**

### CODE REVIEW

File: docs/source/index.md [hunk 1]
```diff
@@ -4,7 +4,104 @@ Computational skills training at the [UCL](https://www.ucl.ac.uk/)
...
+
+Running pose estimation on the SWC HPC system
++++
+Sainsbury Wellcome Centre, UCL <br>
+Ground Floor Lecture Theatre
+
+1pm - 5pm
+:::
+
+[//]: # (:::{grid-item-card} {fas}`file-image;sd-text-primary`  December 5th 2023)
```
Comment (niksirbi) [inline docs/source/index.md:72]: Is this commented out? Sphinx throws warning about the `//`: 

```console
/Users/nsirmpilatze/Code/NIU/software-skills/docs/source/index.md:65: WARNING: Duplicate reference definition: // [myst.duplicate_def]
/Users/nsirmpilatze/Code/NIU/software-skills/docs/source/index.md:67: WARNING: Duplicate reference definition: // [myst.duplicate_def]
/Users/nsirmpilatze/Code/NIU/software-skills/docs/source/index.md:69: WARNING: Duplicate reference definition: // [myst.duplicate_def]
/Users/nsirmpilatze/Code/NIU/software-skills/docs/source/index.md:70: WARNING: Duplicate reference definition: // [myst.duplicate_def]
/Users/nsirmpilatze/Code/NIU/software-skills/docs/source/index.md:72: WARNING: Duplicate reference definition: // [myst.duplicate_def]
/Users/nsirmpilatze/Code/NIU/software-skills/docs/source/index.md:74: WARNING: Duplicate reference definition: // [myst.duplicate_def]
/Users/nsirmpilatze/Code/NIU/software-skills/docs/source/index.md:76: WARNING: Duplicate reference definition: // [myst.duplicate_def]
/Users/nsirmpilatze/Code/NIU/software-skills/docs/source/index.md:78: WARNING: Duplicate reference definition: // [myst.duplicate_def]
/Users/nsirmpilatze/Code/NIU/software-skills/docs/source/index.md:79: WARNING: Duplicate reference definition: // [myst.duplicate_def]
```

File: docs/source/courses/hpc-behaviour.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Running pose estimation on the SWC HPC system
+
+## Overview
+This course will briefly introduce the SWC HPC system and then cover how to run pose estimation at scale on the HPC.
+
+## Prerequisites
+This course will teach HPC usage using video behavioural analysis as an example workflow. Ideally, students should have 
+an understanding of pose estimation tools, e.g. by attending the [Video-based analysis of animal behaviour](./video-analysis) course.
+
+Students should bring their own laptop with Python installed. They should also posess
```
Comment (niksirbi) [inline docs/source/courses/hpc-behaviour.md:10]: Found a typo in my own text
```suggestion
Students should bring their own laptop with Python installed. They should also possess
```

**Review by JoeZiminski — APPROVED** [review]

**Review by niksirbi — APPROVED** [review]

File: docs/source/index.md [hunk 3]
```diff
@@ -4,7 +4,104 @@ Computational skills training at the [UCL](https://www.ucl.ac.uk/)
...
+:link-type: doc
+
+Extracellular electrophysiology analysis
++++
+Sainsbury Wellcome Centre, UCL <br>
+Room TBC
+
+Wed 8th: 9.30am - 12pm & 3.30pm - 5.30pm
+
+Thu 9th: TBC
```
Comment (JoeZiminski) [inline docs/source/index.md:45]:
```suggestion
Thu 9th: 9:30am - 12pm & 3.30pm - 5.30 pm
```

File: docs/source/index.md [hunk 4]
```diff
@@ -4,7 +4,104 @@ Computational skills training at the [UCL](https://www.ucl.ac.uk/)
...
+:::{grid-item-card} {fas}`bolt;sd-text-primary`  November 8th & 9th 2023
+:link: courses/extracellular-analysis
+:link-type: doc
+
+Extracellular electrophysiology analysis
++++
+Sainsbury Wellcome Centre, UCL <br>
+Room TBC
+
+Wed 8th: 9.30am - 12pm & 3.30pm - 5.30pm
```
Comment (JoeZiminski) [inline docs/source/index.md:43]:
```suggestion
Wed 8th: 9.30am - 12pm & 1.30pm - 3.30pm
```

**Pull request merged**